### PR TITLE
Release: 3.2.3

### PR DESCRIPTION
**Pull request by aboedo**
This is the last bump we need in order to publish the hybrid SDKs with Amazon compatibility. It includes the fix to Amazon Live App Testing, and also bumps iOS to the latest version. 
based on #172 

## 3.2.3

- Bump RevenueCat `purchases-ios` to 4.8.0
- Bump RevenueCat `purchases-android` to 5.2.1 (https://github.com/RevenueCat/purchases-hybrid-common/pull/172)

### CODE REVIEW

File: ios/PurchasesHybridCommon/Podfile.lock [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  - RevenueCat (4.8.0)
   - SnapshotTesting (1.9.0)
 
 DEPENDENCIES:
   - Nimble
   - Quick
-  - RevenueCat (= 4.7.0)
-  - "SnapshotTesting (from `git@github.com:pointfreeco/swift-snapshot-testing.git`, branch `main`)"
+  - RevenueCat (= 4.8.0)
+  - SnapshotTesting (from `https://github.com/pointfreeco/swift-snapshot-testing.git`, branch `main`)
```
Comment (NachoSoto) [inline ios/PurchasesHybridCommon/Podfile.lock:11]: This might lead to issues in CI. Did you mean to change it?

**Review by vegaro — APPROVED** [review]
Besides that question that @NachoSoto asked, it looks good to me

**Comment by taquitos** [comment]
Could we wait for `purchases-ios` `4.9.0`? I plan on doing that today

File: ios/PurchasesHybridCommon/Podfile.lock (hunk 1 above)
Comment (aboedo) [reply ios/PurchasesHybridCommon/Podfile.lock:11]: oops, the git thing was failing on my machine so I updated to https locally but missed that it was there in the podfile.lock too. I'll revert this bit

**Comment by taquitos** [comment]
On hold for https://github.com/RevenueCat/purchases-hybrid-common/pull/171

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 3.2.3
+
+- Bump RevenueCat `purchases-ios` to 4.9.0
```
Comment (NachoSoto) [inline CHANGELOG.md:3]: Looks like this isn't updated anymore?

Comment (NachoSoto) [reply CHANGELOG.md:3]: Oh that's already in `main` never mind.

**Review by NachoSoto — APPROVED** [review]

**Pull request merged**